src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrScanner.cs

```diff
@@ -61,6 +65,26 @@ public async Task<List<string>> ScanVmr(CancellationToken cancellationToken)
     }
 
     protected abstract string ScanType { get; }
-    protected abstract Task<IEnumerable<string>> ScanRepository(SourceMapping sourceMapping, CancellationToken cancellationToken);
+    protected abstract Task<IEnumerable<string>> ScanRepository(
+        SourceMapping sourceMapping,
+        string baselineFilesPath,
+        CancellationToken cancellationToken);
+
+    /// <summary>
+    /// Returns a list of files that will be exluded from the scan operation, loaded from the baselineFilesPath
+    /// </summary>
+    protected async Task<IEnumerable<string>> GetBaselineFilesAsync(string repoName, string baselineFilePath)
```

Comment:
I think the name here is misleading - it's not getting the files but exclusion filters for files from the baseline file

Reply:
fixed